docs/source/examples/examples.myst

```diff
@@ -8,11 +8,12 @@ Zero-noise extrapolation with Braket on IonQ backend <zne-braket-ionq.myst>
 Zero-noise extrapolation with Pennylane on IBMQ backends <pennylane-ibmq-backends.myst>
 Zero-noise extrapolation on PyQuil parametric programs <pyquil_demo.ipynb>
 Variational quantum eigensolver improved with zero-noise extrapolation <vqe-pyquil-demo.ipynb>
-Variational circuit mitigation with zero-noise extrapolation <simple_landscape.myst>
+Using ZNE to compute the energy landscape of a variational circuit with Cirq <simple_landscape.myst>
```

Comment:
Do we want to rename the existing one as well to `simple_landscape_cirq.myst`

Reply:
This sounds good to me!